### ISSUE TITLE
Update to newer version of openlayers3.

### ISSUE THREAD

**Issue by bharathnav** [tombatossals/angular-openlayers-directive#260]
HI is there a possibility of updating openlayers dependency to the latest version? It's still pointing to 3.8.2 but the latest available is 3.15.*.

Thanks,
Barrry

**Comment by koihatano** [comment]
+1

BTW, I wonder downgrade from 3.9 to 3.8.2 of openlayers at d192609de49c931f4f5ebed2bd69ec35d58b5588

I try to upgrade to 3.9.0 of openlayers, it passed all unit tests and looks like to work for some example.
Why downgrade openlayers version?

**Comment by TeChn4K** [comment]
Similarly, documentation says to include OL3, but `dist/angular-openlayers-directive.js` do not care, includes his own (3.8.2) and overrides mine (3.16.0)

**Comment by juristr** [comment]
Hi guys, I'm sorry for the delayed responses to issues, but I'm currently quite busy with other stuff. Ya, we should definitely upgrade the ol3 version. To be honest, I'd rather also only reference it as a peer dependency.

But for now, you should be able to use the latest angular-openlayers-directive package and a recent ol3 version, independently from what this package here bundles. Like we're currently using the following combination in production:

```
"angular-openlayers-directive": "~1.12.0",

// installed over npm
"openlayers": "3.13.1"
```

**Comment by juristr** [comment]
This should be fixed by #275. In case please re-open the issue.
Thanks a lot @koihatano for contributing this :+1:
